apps/application-system/form/src/routes/Applications.tsx

```diff
@@ -32,6 +34,58 @@ import { dateFormat } from '@island.is/shared/constants'
 
 import { ApplicationLoading } from '../components/ApplicationsLoading/ApplicationLoading'
 
+interface StateData {
+  tag: {
+    variant: TagVariant
+    label: MessageDescriptor
+  }
+  progress: {
+    variant: 'blue' | 'red' | 'rose' | 'mint'
+  }
+  cta: {
+    label: MessageDescriptor
+  }
+}
+
+const ApplicationStateDisplayedData: Record<ApplicationStatus, StateData> = {
```

Comment:
Would be nice to move that within the core library so it can be exported only once and imported in the form and service portal

Reply:
Yeah I just extracted this to a component that is used both here and in the service-portal